pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: layout.pug
+navigationTitle: Attach Amazon EKS Cluster to Kommander
+title: Attach Amazon EKS Cluster to Kommander
+menuWeight: 7
+excerpt: Attach an existing EKS cluster to Kommander
+beta: true 
+enterprise: false
+---
+
+You can attach any existing Kubernetes clusters to Kommander, then use it to examine and manage those clusters later. The following procedure shows how to attach an existing Amazon Elastic Kubernetes Service (EKS) cluster to Kommander. 
```

Comment:
If we're going to say you can examine and manage these later, we should link to the place to find info on examining and managing them.

Reply:
Yep, good catch!